migrations/versions/0046_organisations_and_branding.py

```diff
@@ -0,0 +1,64 @@
+"""empty message
+
+Revision ID: 0046_organisations_and_branding
+Revises: 0045_billable_units
+Create Date: 2016-08-04 12:00:43.682610
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0046_organisations_and_branding'
+down_revision = '0045_billable_units'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    op.create_table('branding_type',
+        sa.Column('name', sa.String(length=255), nullable=False),
+        sa.PrimaryKeyConstraint('name')
+    )
+    op.create_table('organisation',
+        sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column('colour', sa.String(length=7), nullable=True),
+        sa.Column('logo', sa.String(length=255), nullable=True),
+        sa.Column('name', sa.String(length=255), nullable=True),
+        sa.PrimaryKeyConstraint('id')
+    )
+
+    op.add_column('services', sa.Column('branding', sa.String(length=255)))
+    op.add_column('services', sa.Column('organisation_id', postgresql.UUID(as_uuid=True)))
+    op.add_column('services_history', sa.Column('branding', sa.String(length=255)))
+    op.add_column('services_history', sa.Column('organisation_id', postgresql.UUID(as_uuid=True)))
+
+    op.execute("INSERT INTO branding_type VALUES ('govuk'), ('org'), ('both')")
+    # insert UKVI data as initial test data. hex and crest pulled from alphagov/whitehall
+    op.execute("""INSERT INTO organisation VALUES (
+        '9d25d02d-2915-4e98-874b-974e123e8536',
+        '#9325b2',
+        'ho_crest_27px_x2.png',
```

Comment:
yup - it comes from here: https://www.notifications.service.gov.uk/static/images/email-template/crests/ho_crest_27px_x2.png - file in source code is https://github.com/alphagov/notifications-admin/blob/master/app/assets/images/email-template/crests/ho_crest_27px_x2.png